internal/rhsm/secrets.go

```diff
@@ -74,11 +83,28 @@ func getListOfSubscriptions() ([]subscription, error) {
 	return subscriptions, nil
 }
 
+func getConsumerSecrets() (*ConsumerSecrets, error) {
+	res := ConsumerSecrets{
+		CACert:       "/etc/rhsm/ca/redhat-uep.pem",
+		ConsumerKey:  "/etc/pki/consumer/key.pem",
+		ConsumerCert: "/etc/pki/consumer/cert.pem",
+	}
+
+	if _, err := os.Stat(res.ConsumerKey); err != nil {
+		return nil, fmt.Errorf("no consumer key found")
+	}
+	if _, err := os.Stat(res.ConsumerCert); err != nil {
+		return nil, fmt.Errorf("no consumer cert found")
+	}
+	return &res, nil
+}
+
 // LoadSystemSubscriptions loads all the available subscriptions.
 func LoadSystemSubscriptions() (*Subscriptions, error) {
+	consumerSecrets, _ := getConsumerSecrets()
```

Comment:
Should we log the error and then continue?  Otherwise, maybe we shouldn't be returning the error to begin with?

Reply:
excellent idea